crates/orchestrator/readme.md

```diff
@@ -0,0 +1,93 @@
+# Orchestrator
+
+The Orchestrator crate provides facilities for quickly deploying and benchmarking this codebase in a geo-distributed environment. Please note that it is not intended for production deployments or as an indicator of production engineering best practices. Its purpose is to facilitate research projects by allowing benchmarking of (variants of) the codebase and analyzing performance.
+
+This guide provides a step-by-step explanation of how to run geo-distributed benchmarks on [Amazon Web Services (AWS)](http://aws.amazon.com).
+
+## Step 1. Set up cloud provider credentials
+
+To enable programmatic access to your cloud provider account from your local machine, you need to set up your cloud provider credentials. These credentials authorize your machine to create, delete, and edit instances programmatically on your account.
+
+### Setting up AWS credentials
+
+1. Find your ['access key id' and 'secret access key'](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds).
+2. Create a file `~/.aws/credentials` with the following content:
+
+```text
+[default]
+aws_access_key_id = YOUR_ACCESS_KEY_ID
+aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
+```
+
+Do not specify any AWS region in that file, as the scripts need to handle multiple regions programmatically.
+
+## Step 2. Specify the testbed configuration
+
+Create a file called `settings.json` that contains all the configuration parameters for the testbed deployment. You can find an example file at `./assets/settings.json` with the following content:
+
+```json
+{
+  "testbed_id": "alberto-mysticeti",
+  "cloud_provider": "aws",
+  "token_file": "/Users/alberto/.aws/credentials",
+  "ssh_private_key_file": "/Users/alberto/.ssh/aws",
+  "regions": [
+    "us-east-1",
+    "us-west-2",
+    "ca-central-1",
+    "eu-central-1",
+    "ap-northeast-1",
+    "eu-west-1",
+    "eu-west-2",
+    "ap-south-1",
+    "ap-southeast-1",
+    "ap-southeast-2"
+  ],
+  "specs": "m5d.8xlarge",
+  "repository": {
+    "url": "http://github.com/mystenlabs/project-mysticeti.git",
```

Comment:
Can this be the Sui repo or any other repo?

Reply:
it theory any other repo - there are some additional processes that run though (ex generate committee) which might not apply for other repos.